components/kernels/dotnet.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import * as React from "react";
+
+import Kernel from "./kernel";
+import SyntaxHighlighter from "react-syntax-highlighter";
+import { github } from "react-syntax-highlighter/styles/hljs";
+import { ContentSection } from "../content-section";
+
+const install = `dotnet tool install -g Microsoft.dotnet-interactive`;
+const installKernelSpecs = `dotnet interactive jupyter install`;
+
+export default () => (
+  <ContentSection>
+    <ContentSection.Pane full>
+      <Kernel
+        displayName=".NET Interactive"
+        repository="https://github.com/dotnet/interactive"
+        installURL="https://github.com/dotnet/interactive/blob/master/NotebooksLocalExperience.md"
+        logo="/static/dotnet-icon.png"
+      >
+        <h3>Installation</h3>
+        <p>
+        Install <a href="https://dotnet.microsoft.com/download">.NET Core sdk</a> to
+              your system.
+        </p>
+        <div className="columns">
+          <div className="column">
+            <h4>With .NET Core sdk 3.1 installed, from the terminal install the global tool</h4>
```

Comment:
Nit: Capitalize SDK.

Reply:
Also, let's rephrase this.

"First, install the .NET Core SDK. Then use the command line tool to install the global tool."